weld/llvm.rs

```diff
@@ -901,7 +903,22 @@ impl LlvmGenerator {
                     self.prelude_code.add(&name_replaced);
                     self.prelude_code.add("\n");
 
-                    // Supports vectorization, so splice in the vector extensions.
+                    // Generate comparison functions, using a special template for vec[i8].
+                    if *elem.as_ref() == Scalar(ScalarKind::I8) {
```

Comment:
It won't be correct for signed integers. Probably won't work well on floats/doubles either but I'm not 100% sure.